model/detailDao.js

```diff
@@ -0,0 +1,18 @@
+const myDataSource = require("../util/dataSource");
+
+const getProducts = async (productId) => {
+  const data = await myDataSource.query(
+    `
+  SELECT * , 
+  pt.packing_types 
+  FROM products AS p
+  INNER JOIN packing_types pt ON p.packing_type_id = pt.id
+  WHERE p.id = ?
+  `,
+    [productId]
+  );
+
+  return data;
+};
```

Comment:
WEHRE 절에서 pk 값을 기준으로 필터링하고 있네요. 하나의 상품정보를 가져오는 엔드포인트이므로 함수명을 `getProducts()` -> `getProduct()`로 수정해주세요~

```suggestion
const getProduct = async (productId) => {
  const data = await myDataSource.query(
    `
  SELECT * , 
  pt.packing_types 
  FROM products AS p
  INNER JOIN packing_types pt ON p.packing_type_id = pt.id
  WHERE p.id = ?
  `,
    [productId]
  );

  return data;
};
```